### ISSUE TITLE
Color contrast ratio of the focus border on the code is less than 3:1 ratio 

### ISSUE THREAD

**Issue by cwebster-99** [microsoft/PTVS#7856]
ref https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1988152

**Environment Details:**
Application Name: Accessibility Testing for Python in VS
Edition Windows 11 Enterprise
Version 22H2
OS build 22621.3155
Windows Feature Experience Pack 1000.22684.1000.0
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.9.2

**Repro Steps:**
Open visual studio.
TAB till Create a new Project and press ENTER key.
Create a "project" dialog will open.
Search "Flask Web" project and select it.
TAB till next button and press ENTER key.
Configure your new project screen will open.
Provide project name.
TAB to create button and press enter.
Project will be created.
Open views.py and write/edit code in the code editor.
Run Accessibility insights for windows.
Now observe that Color contrast ratio of the focus border on the code is less than 3:1 ratio i.e., 1.196:1

**Actual:**
Color contrast ratio of the focus border on the code is less than 3:1 ratio i.e., 1.196:1.

**Expected:**
Color contrast ratio of the focus border on the code should be greater than or equal to 3:1 and should not be less than 3:1.

**User Impact:**
Low vision user will face difficulties if the focus border is not visible on the code if it is less than 3:1.
